example.config.yml

```diff
@@ -168,6 +169,10 @@ installed_extras:
 # Add any extra apt or yum packages you would like installed.
 extra_packages: []
 
+# `newrelic` must be in installed_extras for this to work.
+# See all vars: https://github.com/weareinteractive/ansible-newrelic#variables
+newrelic_license_key: yourkey
```

Comment:
I'd rather leave this out by default, and have a new docs page under 'Extra software and setup' that describes how to enable New Relic support. I have a feeling that the number of people who will use New Relic with Drupal VM is fairly small (percentage-wise), so I'd rather leave these three lines out of the example config.yml